test/DebugInfo/debug_compilation_dir.swift

```diff
@@ -0,0 +1,11 @@
+// RUN: %empty-directory(%t)
+// RUN: mkdir -p %t
+// RUN: cd %t
+// RUN: cp %s .
```

Comment:
you can drop the top 4 lines here since this test will not produce any disk output